### PR TITLE
Fix coveralls

### PR DESCRIPTION
**Pull request by drcrallen**
Fixes coveralls issues
https://coveralls.io/jobs/10696579

### CODE REVIEW

**Comment by fjy** [comment]
:+1:

File: benchmarks/pom.xml [hunk 1]
```diff
@@ -118,46 +118,6 @@
         </executions>
       </plugin>
     </plugins>
-    <pluginManagement>
```
Comment (xvrl) [inline benchmarks/pom.xml]: I think those plugin versions should be left in, they are generated by jmh when setting up the pom

Comment (drcrallen) [inline benchmarks/pom.xml]: ok will revert

**Comment by nishantmonu51** [comment]
:+1:

**Pull request merged**